### ISSUE TITLE
hilarious

### ISSUE THREAD

**Issue by environment-dev** [ian-ok/ventry.host#1]
hilarious

**Comment by saintly2k** [comment]
what a retard, whoever did this is even worse than me in php

**Comment by ian-ok** [comment]
> what a retard, whoever did this is even worse than me in php

Yeah, the code is dogshit, he reconnects to the db every time you go to a new page. Also the email password reset page is highly SQL injectable

**Comment by xbdmdev** [comment]
lol

**Comment by xbdmdev** [comment]
man down: https://ventry.host/

**Comment by saintly2k** [comment]
finally

**Comment by xbdmdev** [comment]
he gave up his shitty codding skill and moved to horizon.pics hahah

**Comment by saintly2k** [comment]
XDDDD

**Comment by clynt707** [comment]
This source makes me crazy!!!! 
I do the torture to myself and make the source clean

**Comment by zSatur** [comment]
> he gave up his shitty codding skill and moved to horizon.pics hahah

nah he quit cause of this thing
horizon just moved the domain thing

**Comment by ian-ok** [comment]
Meow
